tools/snitch/snitcher/docker/windows/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/windows:1809-amd64 AS builder
```

Comment:
What is the change to this line?

Reply:
Probably a new line character at the end of line. I suspect this used to be the Unix EOL.